File: src/Lession5/Ex3.java
package Lession5;

public class Ex3 {
    static void main() {
        int[] array = new int[] {7, 2, 9, 4, 0, 6, 3, 8, 1, 5, 3, 2, 7, 4, 9, 1, 0, 6, 3, 8};

        int max = array[0];
        int max_first_index = 0;

        for (int i = 0; i < array.length; i++) {
            if (array[i] > max) {
                max = array[i];
                max_first_index = i;
            }
        }

        System.out.println("First max at index: " + max_first_index);
    }
}

File: src/Lession5/Ex3.kt
package Lession5

fun main() {
    val array = intArrayOf(3, 7, 1, 9, 4, 0, 8, 2, 5, 6, 5, 3, 1, 7, 4, 9, 2, 0, 6, 8)

    var first_max_index = 0
    var max = array[0]

    for (i in 1 until array.size) {
        if (array[i] > max) {
            max = array[i];
            first_max_index = i
        }
    }

    println("First max at index: $first_max_index")
}
